jest.config.ts

```diff
@@ -4,8 +4,7 @@ export default {
   errorOnDeprecated: true,
   moduleDirectories: ['node_modules', '<rootDir>'],
   preset: 'ts-jest/presets/js-with-ts',
-  restoreMocks: true,
   roots: ['<rootDir>/src'],
-  setupFilesAfterEnv: ['<rootDir>/.jest/setup/jestDomExpects.ts'],
-  testEnvironment: 'jsdom',
+  testEnvironment: 'node',
+  testRegex: '(/__tests__/.*|(\\.|/)test)\\.tsx?$',
```

Comment:
should that be in the example project config already?

Reply:
No, the default is fine 99% of the time. It catches and wants to run our mock as test suites, that's why we're more explicit here.